tests/test_generic/tests/test_xml.py

```diff
@@ -46,18 +52,46 @@ def _check_files_in_path(self, module):
                     else:
                         manifest_content = content
                     continue
+
                 self._check_xml_style(content, module, file_name)
                 self._check_update_status(content, file_name)
                 self._check_useless_models(content, file_name)
                 self._check_useless_fields_on_models(content, file_name)
                 self._check_knowledge_article_is_published(content, file_name)
                 self._check_duplicate_records(content, file_name)
                 self._check_website_published_false(module, file_name)
+                self._check_static_files_usage_in_xml(content, in_use_files)
                 if root.split('/')[-1] == 'data':
                     self._check_is_published_false(module, file_name)
                     if not is_studio_required:
                         is_studio_required = self._check_studio(content, file_name)
         self._check_manifest(manifest_content, is_studio_required)
+        print(in_use_files)
+        in_use_files = set(file.lstrip('/') for file in in_use_files)
+        for file in static_files - in_use_files:
+            _logger.warning(f"Unused static file {file}.")
+        for file in in_use_files - static_files:
+            _logger.warning(f"No reference found for this file: {file}.")
+
+    def _check_static_files_usage_in_xml(self, content, in_use_files):
+        tree = etree.fromstring(content.encode('utf8'))
+        for element in tree.iter():
+            file_names = set(
+                element.attrib.get(key) for key in ['file', 'src', 'data-original-src']
+            )
+            if element.text:
+                if element.get("name") == "cover_properties":
+                    file_names.update(re.findall(r"url\(['\"]?([^'\")]+)['\"]?\)", element.text))
+                file_names.update(re.findall(r'src="([^"]+)"', element.text))
+
+            in_use_files.update(
+                {
+                    file
+                    for file in file_names
+                    if file
+                    and not file.startswith(('web', '/web', 'https', '/unsplash', 'sale_timesheet'))
```

Comment:
I'd add the file in static of the industry instead of escaping the test here
```suggestion
                    and not file.startswith(('web', '/web', 'https', '/unsplash'))
```